config/routes.rb

```diff
@@ -518,6 +527,9 @@
           delete :unlink
         end
       end
+      # TODO: This is for now only added to be able to create a link
+      # There is no controller behind this.
+      resources :project_life_cycles, only: %i[index]
```

Comment:
Not sure about that. Admittedly, it is semantics. To me using project_life_cycles is distinctive enough and I'd expect that the distinction between step_definitions and steps is only relevant from a technical perspective. To the user I would assume it is actually somewhat of the same. So with the url within the project being `projects/:id/settings/life_cycle` it makes sense to me that the url within the administration would be `admin/settings/project_life_cycles`. Ideally, I would go for `admin/projects/life_cycles` but that is a different story.

Reply:
I agree that having all words of life cycle step definitions is to much, though we can use `path: 'life_cycle'` option to have internally complete identification of path and sane url

Reply:
I took up the idea of using a path option but settled for `project_life_cycles` given that I could imagine other life cycles as well. Again, ideally I would go for `admin/projects/life_cycles` but since the other project specific routes aren't in line with that, I think it prudent to keep the `project_` prefix.